ui/public/index.html

```diff
@@ -10,7 +10,7 @@
     <!-- Google Platform JS for authentication
       This puts `gapi` into the global namespace
     -->
-    <script src="https://apis.google.com/js/platform.js"></script>
+    <script src="https://accounts.google.com/gsi/client" async defer></script>
```

Comment:
What is this change doing?

Reply:
ah, sorry, I was trying to size https://precisionmedicineinitiative.atlassian.net/browse/RW-8907, did not realize I was using this branch. Reverted.